jax/_src/scipy/fft.py

```diff
@@ -125,6 +125,39 @@ def dctn(x: Array, type: int = 2,
     - :func:`jax.scipy.fft.dct`: one-dimensional DCT
     - :func:`jax.scipy.fft.idct`: one-dimensional inverse DCT
     - :func:`jax.scipy.fft.idctn`: multidimensional inverse DCT
+
+  Example:
+
+    ``jax.scipy.fft.dctn`` computes the transform along both the axes by dafault
+    when ``axes`` argument is ``None``.
+
+    >>> x = jax.random.normal(jax.random.key(0), (3, 3))
+    >>> jax.scipy.fft.dctn(x)
+    Array([[-5.0406537 , -7.5423923 , -3.2589488 ],
+           [ 0.83180463,  3.640377  , -4.027303  ],
+           [ 0.11503673, -0.73313326,  3.7358375 ]], dtype=float32)
+
+    When ``s=[2]``, dimension of the transform along ``axis 0`` will be ``2``
+    and dimension along ``axis 1`` will be same as that of input.
+
+    >>> jax.scipy.fft.dctn(x, s=[2])
+    Array([[-2.9185386 , -2.6821108 , -5.7430754 ],
+           [ 0.42092603,  0.9677744 ,  0.997496  ]], dtype=float32)
+
+    When ``s=[2]`` and ``axes=[1]``, dimension of the transform along ``axis 1`` will
+    be ``2`` and dimension along ``axis 0`` will  be same as that of input.
+    Also when ``axes=[1]``, transform will be computed only along ``axis 1``.
+
+    >>> jax.scipy.fft.dctn(x, s=[2], axes=[1])
+    Array([[-0.21575952, -0.89992356],
+           [-0.5741661 , -1.6780441 ],
+           [-2.524435  , -0.11382911]], dtype=float32)
+
+    When ``s=[2, 4]``, shape of the transform will be ``(2, 4)``.
+
+    >>> jax.scipy.fft.dctn(x, s=[2, 4])
+    Array([[-2.9185386 , -2.4928849 , -4.20934   , -5.572744  ],
+           [ 0.42092603,  0.7851271 ,  1.1611955 ,  0.80011123]], dtype=float32)
```

Comment:
We may need to skip these examples for doctests, because floating point outputs may not be bitwise equivalent across systems.

Reply:
Another option is to explicitly round the input, similar to these tests: https://github.com/google/jax/blob/9e1a4e8d743ba584ed755816b2687029dda1d083/jax/_src/scipy/linalg.py#L1738-L1739

Reply:
Thanks for the suggestion. Printed the outputs by rounding with precision=2.